README.md

```diff
@@ -126,13 +126,66 @@ The following is an example `membersUpdate` event received by listeners when a u
 ]
 ```
 
+### Members
+
+Members is used to build avatar stacks and find out which members are online within a space.
+
+```ts
+// Subscribe to all member events in a space by passing the 'update' event
+space.members.subscribe('update', (memberUpdate) => {
+  console.log(memberUpdate);
+});
+
+// Subscribe to when a member enters the space
+space.members.subscribe('enter', (memberJoined) => {
+  console.log(memberJoined);
+});
+
+// Subscribe to when a member leaves the space
+space.members.subscribe('leave', (memberLeft) => {
+  console.log(memberLeft);
+});
+```
+
+The following is an example `memberUpdate` event received by listeners:
+
+```json
+{
+  "clientId": "clemons#142",
+  "connectionId": "hd9743gjDc",
+  "isConnected": true,
+  "lastEvent": {
+    "name": "enter",
+    "timestamp": 1677595689759
+  },
+  "location": null,
+  "profileData": {
+    "username": "Claire Lemons",
+    "avatar": "https://slides-internal.com/users/clemons.png"
+  }
+}
+```
+
+Members has methods to get the current snapshot of member state:
+
+```ts
+// Get all members in a space
+const allMembers = space.members.getAll();
+
+// Get your own member object
+const myMemberInfo = space.members.getSelf();
+
+// Get everyone else's member object but yourself
+const othersMemberInfo = space.members.getOthers();
```

Comment:
```suggestion
// Get all members in a space
const allMembers = await space.members.getAll();

// Get your own member object
const myMemberInfo = await space.members.getSelf();

// Get everyone else's member object but yourself
const othersMemberInfo = await space.members.getOthers();
```

(these are now async since https://github.com/ably-labs/spaces/pull/116).